current/ReactWindows/ReactNative/Modules/DeviceInfo/DeviceInfoModule.cs

```diff
@@ -161,16 +161,13 @@ private void OnOrientationChanged(DisplayInformation displayInformation, object
                     break;
             }
 
-            if (name != null)
-            {
-                Context.GetJavaScriptModule<RCTDeviceEventEmitter>()
-                    .emit("namedOrientationDidChange", new JObject
-                    {
-                        { "name", name },
-                        { "rotationDegrees", degrees },
-                        { "isLandscape", isLandscape },
-                    });
-            }
```

Comment:
Sorry if the description in the issue wasn't clear.  The code you're modifying is for the "current" version of react-native-windows.  This is the older version of react-native-windows, which will soon be deprecated and replaced by "vnext", a new, C++ based implementation.  

From the looks of it, the code here in "current" already implements this correctly and doesn't need any changes.  The issue is tracking adding this support to "vnext".  So what's needed is to implement something similar to this in the vnext version of this module ([https://github.com/microsoft/react-native-windows/blob/master/vnext/ReactUWP/Modules/DeviceInfoModule.cpp](https://github.com/microsoft/react-native-windows/blob/master/vnext/ReactUWP/Modules/DeviceInfoModule.cpp)), and to test it out to make sure it works.

To get started with making changes to vnext, please read the Getting Started guide and follow the steps for vnext (see [https://github.com/microsoft/react-native-windows/blob/master/vnext/docs/GettingStarted.md](https://github.com/microsoft/react-native-windows/blob/master/vnext/docs/GettingStarted.md)).  Then follow the steps for building the vnext repo  (see:   [https://github.com/microsoft/react-native-windows/blob/master/vnext/docs/BuildingRNW.md](https://github.com/microsoft/react-native-windows/blob/master/vnext/docs/BuildingRNW.md)).  Once you have it building, let us know if you need more pointers on what the code changes should look like (basically it will look like this code, but ported to C++).